test/test-project/src/test/bottomBar/views-test.ts

```diff
@@ -78,15 +81,25 @@ import { BottomBarPanel, OutputView, TerminalView, VSBrowser, Workbench } from '
             expect(text).not.empty;
         });
 
-        it('executeCommand works', async () => {
+        it('executeCommand works', async function () {
             const command = `${process.platform === 'win32' ? 'start-sleep -s' : 'sleep'} 2`;
             await terminal.executeCommand(command, 3000);
         });
 
-        it('newTerminal opens a new term channel', async () => {
+        it('newTerminal opens a new term channel', async function () {
             await terminal.newTerminal();
             const channel = await terminal.getCurrentChannel();
             expect(channel).equals(`2: ${terminalName}`);
         });
     });
+
+    async function selectChannel(name: string): Promise<void> {
```

Comment:
Would not be better to put this logic into view.selectChannel?

Reply:
nice catch! Thank you I forget to move logic to output view